packages/components/src/radio/src/Radio.css

```diff
@@ -2,45 +2,45 @@
     display: inline-flex;
     align-items: center;
     cursor: pointer;
-    line-height: var(--o-ui-lh-1);
     max-width: 100%;
     width: max-content;
     height: max-content;
-    color: var(--o-ui-text-alias-primary);
+    color: var(--hop-neutral-text);
 }
 
 /* BUTTON */
 .o-ui-radio-button {
     display: inline-block;
-    background-color: var(--o-ui-bg-alias-surface);
-    border-radius: var(--o-ui-br-circular);
-    border: 1px solid var(--o-ui-b-alias-mid-break);
+    background-color: var(--hop-neutral-surface);
+    border-radius: var(--hop-shape-circle);
+    border: 1px solid var(--hop-neutral-border);
     position: relative;
-    width: var(--o-ui-sz-3);
-    height: var(--o-ui-sz-3);
+    width: 1.5rem;
+    height: 1.5rem;
     flex-shrink: 0;
 }
 
 .o-ui-radio-button::before {
+    --o-ui-radio-button-size: 0.75rem;
     content: "";
     display: block;
-    background-color: var(--o-ui-bg-alias-accent);
-    border-radius: var(--o-ui-br-circular);
+    background-color: var(--hop-neutral-icon-active);
+    border-radius: var(--hop-shape-circle);
     transform: scale(0);
     transition: all var(--o-ui-easing-duration-1) var(--o-ui-easing-productive);
     position: absolute;
     top: 50%;
     left: 50%;
-    width: var(--o-ui-sz-3);
-    height: var(--o-ui-sz-3);
-    margin-top: calc(var(--o-ui-sz-3) * -1 / 2);
-    margin-left: calc(var(--o-ui-sz-3) * -1 / 2);
+    width: var(--o-ui-radio-button-size);
+    height: var(--o-ui-radio-button-size);
+    margin-top: calc(var(--o-ui-radio-button-size) * -1 / 2);
+    margin-left: calc(var(--o-ui-radio-button-size) * -1 / 2);
```

Comment:
The dot seems off centered to me:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/4177ff19-85c1-491d-8bda-4da3d7d37b17)
I would use translate(-50%, -50%) and see if that helps.

Reply:
You may also have to set transform-origin: top left;

Reply:
It must be an optic effect, I copied the render in Figma to validate and it looks perfect. If you move the original transform, you'll lose the animation effect when you click.

![Capture d’écran, le 2023-10-20 à 13 48 33](https://github.com/gsoft-inc/wl-orbiter/assets/2379339/412b3d66-3da1-4e88-80f0-5d8a54bf9129)